saml-handler/pom.xml

```diff
@@ -0,0 +1,378 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor
+    license agreements. See the NOTICE file distributed with this work for additional
+    information regarding copyright ownership. The ASF licenses this file to
+    you under the Apache License, Version 2.0 (the "License"); you may not use
+    this file except in compliance with the License. You may obtain a copy of
+    the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required
+    by applicable law or agreed to in writing, software distributed under the
+    License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+    OF ANY KIND, either express or implied. See the License for the specific
+    language governing permissions and limitations under the License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.apache.sling</groupId>
+    <artifactId>sling</artifactId>
+    <version>37</version>
+    <relativePath />
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.sling.auth</groupId>
+  <artifactId>saml2</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+  <name>SAML2 Service Provider</name>
+
+  <properties>
+    <opensaml.version>3.4.3</opensaml.version>
+    <sling.host>localhost</sling.host>
+    <sling.port>8080</sling.port>
+    <sling.user>admin</sling.user>
+    <sling.password>admin</sling.password>
+  </properties>
+
+  <packaging>bundle</packaging>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
```

Comment:
I would suggest moving to the bnd-maven-plugin, as all of Sling is doing that and it makes it simpler for us to maintain and evolve the module.

Reply:
I made the change to bnd-maven-plugin